### PR TITLE
[doc] update dtensor readme

### PR DESCRIPTION
**Pull request by feifeibear**
I fixed some import erros in readme of dtensor.

cc @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @H-Huang @kwen2501 @awgu

### CODE REVIEW

**Review by wanchaol — APPROVED** [review]
Thanks for the fix!

File: torch/distributed/_tensor/README.md [hunk 1]
```diff
@@ -47,28 +47,43 @@ Here are some basic DistributedTensor API examples that showcase:
...
+world_size = torch.distributed.get_world_size()
 # construct a device mesh with available devices (multi-host or single host)
 device_mesh = DeviceMesh(device_type="cuda", [0, 1, 2, 3])
 # if we want to do row-wise sharding
 rowwise_placement=[Shard(0)]
 # if we want to do col-wise sharding
 colwise_placement=[Shard(1)]
 # distributed tensor returned will be sharded across the dimension specified in placements
-distributed.empty((8, 12), device_mesh=device_mesh, placements=rowwise_placement)
+dist_tensor = DTensor(
```
Comment (wanchaol) [inline torch/distributed/_tensor/README.md:60]: Can we please not change those code examples? The code examples in README are API reference and some of the features not implemented yet, i.e distributed.empty. DTensor(torch.empty()) is not a public API and should not be used here.

**Review by wanchaol — CHANGES_REQUESTED** [review]

File: torch/distributed/_tensor/README.md (hunk 1 above)
Comment (feifeibear) [reply torch/distributed/_tensor/README.md:60]: I see. I have reverted these back!

**Review by wanchaol — APPROVED** [review]
Shipit

**Pull request closed without merging**